Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -708,54 +614,6 @@ extension ChannelPipeline.SynchronousOperations {
 
         return try handler.syncAsyncStreamMultiplexer(continuation: continuation, inboundStreamChannels: inboundStreamChannels)
     }
-
-    /// Configures a `ChannelPipeline` to speak HTTP/2 and wraps any created inbound stream channels in `NIOAsyncChannel`s.
```

Comment:
Why are we removing this variant with this change? It's not related to protocol negotiation.